packages/paste-website/src/pages/components/chat-log/index.mdx

```diff
@@ -340,6 +340,18 @@ You can push or pop a chat based on an action or event. In this example it's bas
   {chatLoggerExample}
 </LivePreview>
 
+## Composition Notes
+
+Keep content as concise as possible given the space constraints.
+
+### Error messages
+
+To convey an error related to whether a message was sent, use Help Text inside the Message Meta. Limit the message to a single sentence and focus on how the user can resolve the issue. Offer a [link-style retry button](/components/button#link-style-button) when applicable.
+
+To convey an error related to other actions taken in the Chat Log, like a file upload validation error, use an [Alert](/components/alert) at the top of the individual chat window.
```

Comment:
Does this make sense / possible with paste? i'm thinking of a message that shows at the top of the chat window.

Reply:
This would use a Toast instead since it's in response to a user action, but this guideline still holds for Toast.

Reply:
OK. Will update this guidance.